tests/compile-fail/lint_pass.rs

```diff
@@ -0,0 +1,31 @@
+#![feature(plugin)]
+#![feature(rustc_private)]
+#![plugin(clippy)]
+
+#![deny(lint_without_lint_pass)]
+//~^ NOTE:
+
+#[macro_use] extern crate rustc;
+
+use rustc::lint::{LintPass, LintArray};
+
+declare_lint! { GOOD_LINT, Warn, "good lint" }
+declare_lint! { MISSING_LINT, Warn, "missing lint" }
+//(DISABLED)~^ ERROR: the lint MISSING_LINT is not added to any LintPass</del>
+
+// FIXME: Either the JSON output or compiletest-rs is wrong. The error above is
+//        not captured.
```

Comment:
Ah no… I see, the JSON has `"file_name":"<rustc macros>"` as the span. compiletest-rs thinks that's a different file and just ignores it. See if you can get another span for that.